servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/PubFirstOrError.java

```diff
@@ -20,6 +20,8 @@
 import java.util.NoSuchElementException;
 import javax.annotation.Nullable;
 
+import static io.servicetalk.concurrent.api.SubscriberApiUtils.wrapNull;
```

Comment:
Unused import.

Reply:
You may have been confused because the new method name is the same as before in the parent class. This import is used.